### ISSUE TITLE
Can we have several environments for one Salesforce org ?

### ISSUE THREAD

**Issue by FabienTaillon** [forcedotcom/try-code-builder-feedback#181]
**General feedback**
Once I created a Code Builder env, I can't create a second one.
We have some orgs with several repositories, how are we supposed to work with them with only env ?
I couldn't find a way in Code Builder to switch to another repo.

**Comment by msrivastav13** [comment]
You cannot have multiple Code Builder environments at this point, but you can certainly switch between different repositories and connect multiple salesforce orgs and functions environments.

Use the three lines menu at the top left and choose file and look for your other projects.

<img width="1766" alt="Screen Shot 2022-08-12 at 2 34 04 PM" src="https://user-images.githubusercontent.com/2276156/184422239-3685d172-d9f6-445d-9344-ad1732e7f8d5.png">

Maybe we need to document this better @smaddox-sf

**Comment by FabienTaillon** [comment]
@msrivastav13 I was able to open home/codebuilder, git clone my other repo, and then open this folder.
Is it what I'm supposed to do ? Consider this as a remote hard drive and git clone stuff in there ?

Looks like I can navigate to a lot of folders/files, should I really be able to access all of theses ?

<img width="612" alt="folder" src="https://user-images.githubusercontent.com/2465122/184431636-ffaaca65-9dab-4930-a1b4-7a953473e4c6.png">

**Comment by msrivastav13** [comment]
Yes, I think that's correct. You might even have your folder in there where you can have all your Projects.

Note I don't recommend editing any system files or folders unless you know what you are doing!

**Comment by FabienTaillon** [comment]
Ok ! So that was definitely unclear to me. That feels weird to have a full setup from Salesforce, coming to Code Builder with everything setup, in a limited environment where you can't install extensions, but then have to manually go to the file system to get a new repo.
